demo/train_yolov3_on_ipu.sh

```diff
@@ -0,0 +1,4 @@
+exp_name=yolov3_d53_320_273e_coco_ipu
```

Comment:
This sh script is not very suitable to put here. What do you think about putting it in `configs/yolo`?  Relevant content descriptions can be placed in README